plugins/search-react/src/components/SearchFilter/SearchFilter.tsx

```diff
@@ -15,6 +15,8 @@
  */
 
 import React, { ReactElement, ChangeEvent } from 'react';
+import _ from 'lodash';
```

Comment:
```suggestion
import isEmpty from 'lodash/isEmpty';
```

Preferring this for tree shaking

Reply:
done